app/Policies/UserPolicy.php

```diff
@@ -30,6 +32,15 @@ public function accessAdmin(User $user): bool
         return $user->hasAnyRole(['coordinator', 'superuser', 'super-admin', 'coach', 'cooperation-admin']);
     }
 
+    public function sendUserInformationToEconobis(Account $account, User $user)
+    {
+        $userRelatedToEconobis = app(UserService::class)->forUser($user)->isRelatedWithEconobis();
```

Comment:
Isn't it just easier to return this statement, since it returns/is evaluated as a boolean?